lib/active_job/scheduled.rb

```diff
@@ -0,0 +1,7 @@
+module ActiveJob::Scheduled
+  extend ActiveSupport::Concern
+
+  def scheduled_enqueue_delayed?
```

Comment:
I don't know if I like this name; it feels a bit too long/verbose.

`delayed?` on the other hand might be too generic

Reply:
Yes, I agree on both... 

I'm also not sure that this method should live on `ActiveJob` 🤔  It's just a view concern here, so it feels a bit premature to include it all the way into Active Job as an extension. It seems the only reason we want it as a job method might be the tests, but these could be turned into a controller test instead. 

I think I'd do that, and solve the issue with this method being too long/verbose. I'd have this as a view helper, where you wouldn't need to check `status == :scheduled` (since you'd be calling it from `scheduled/_job`).

Reply:
👀 Great idea! I'll update the PR to refactor it as a view helper and a controller test

Reply:
Thank you! 🙏